VideoWeb/VideoWeb/ClientApp/src/app/shared/force-play-video.directive.ts

```diff
@@ -40,9 +40,9 @@ export class ForcePlayVideoDirective implements OnInit, OnChanges {
 
     private updateMute() {
         if (!!this.mute) {
-            this.renderer.setAttribute(this.videoElement, 'muted', 'true');
+            this.videoElement.muted = true;
         } else {
-            this.renderer.setAttribute(this.videoElement, 'muted', 'false');
+            this.videoElement.muted = false;
```

Comment:
It would be good if we could use this.renderer.setProperty method instead, we didn't have the chance to test if that would actually work today though.